content/concepts/modules.md

```diff
@@ -18,8 +18,8 @@ webpack builds on lessons learned from these systems and applies the concept of
 
 In contrast to Node.js modules, webpack _modules_ can express their _dependencies_ in a variety of ways. A few examples are:
 
-* A JavaScript `require()` statement
 * An ECMAScript2015 `import` statement
```

Comment:
AFAIK ES6 is officially wrong and ES2015 is the correct version number. They switched from numbers to years.